include/RAJA/pattern/kernel/Param.hpp

```diff
@@ -22,19 +22,13 @@
 #include "RAJA/config.hpp"
 
 #include "RAJA/pattern/kernel/internal.hpp"
+#include "RAJA/pattern/params/params_base.hpp"
 
 #include <iostream>
 #include <type_traits>
 
 namespace RAJA
 {
-namespace internal
-{
-
-struct ParamBase
-{};
-
-}  // end namespace internal
```

Comment:
An alternative would be to make an alias here instead of changing it everywhere.